test/jdk/tools/launcher/modules/illegalaccess/IllegalAccessTest.java

```diff
@@ -24,34 +24,14 @@
 /**
  * @test
  * @requires vm.compMode != "Xcomp"
```

Comment:
I think `@requires vm.compMode != "Xcomp" was added because the test took too long with -Xcomp. I think it can be dropped now.

Reply:
Dropped.